ui/partials/perf/comparechooserctrl.html

```diff
@@ -16,7 +20,11 @@
           <label for="new-project-selector">Project</label>
           <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects"/>
           <label for="new-revision-input">Revision</label>
-          <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
+            <div class="form-group has-error">
+            <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
+            <br/>
```

Comment:
Oh, and on that line 25, I'm not sure about closing that `<br/>`, I defer to @wlach. I see in the google style guide for HTML5, they indicate void tags should not receive one.

*"Although fine with HTML, do not close void elements, i.e. write \<br>, not \<br /*>."*
http://google-styleguide.googlecode.com/svn/trunk/htmlcssguide.xml#HTML_Style_Rules > Document Type